sway-lib-std/src/codec.sw

```diff
@@ -5347,7 +5347,7 @@ use ::ops::*;
 
 pub fn contract_call<T, TArgs>(
     contract_id: b256,
-    method_name: str,
+    method_name: raw_slice,
```

Comment:
Strictly speaking this is a breaking change on a `pub fn`. To my knowledge, compiler is the only user. Are there any documented use cases where someone can actually use `contract_call`? I am not aware of it so this breaking change looks ok to me.

Reply:
I don't see a reason for this to be used. And on top of that, the `new_encoding` is still on experimental, so I think this is fine to break.